src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20160512_status_change_log.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="dummy" author="davidan" id="status_change_log">
+
+        <createTable tableName="STAT_WORKFLOW_STATUS">
+            <column autoIncrement="true" name="id" type="BIGINT">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="workflow_id" type="INT">
+                <constraints nullable="false"/>
+            </column>
```

Comment:
I am intentionally NOT making these foreign keys. As submissions/workflows are deleted, I do NOT want these deleted; it's worth keeping history over time even for deleted submissions/workflows. Happy to hear objections to this.

Yes, I realize that we would not be able to retrieve method, entity, user, etc. info for any deleted submissions/workflows; we can still keep the status change timing.